src/AzureAuth/OutputMode.cs

```diff
@@ -29,5 +29,10 @@ public enum OutputMode
         /// The none.
         /// </summary>
         None,
+
+        /// <summary>
+        /// The Security Identifier.
+        /// </summary>
+        SID,

```

Comment:
I recall the spec included having a way to get your SID, but I'm not sure we want to make it a formal output mode, I was thinking perhaps, the SID could be included in the "info" output. I realize that requires authenticating though, so I'm curious what others think about this as the medium for which to get a SID.

Here's what I don't like:
The goal of allowing the user to see their SID easily is for them to tell it to us, when they need us to review their telemetry. In order to use this easily, one would have to run the entire main (to be `aad`) command, with all the requires args, **or**, use an alias. As we branch out, making a consistent way to get this I think will be a much better user experience. 

I think an easier UX would be to include the SID in the output of the new `info` command. And the auth to do it, could be to AzureDevops - since we are going to be hardcoding the VS and ADO Guids into the code base for PATs anyway.

Reply:
We could also see if the SID is available on Windows directly from the OS, and only use auth to get it, on Mac and Linux.

Reply:
On Windows I didn't get a reliable way to get the SID. Although some command like `wmic.exe useraccount where "localaccount=true" get name,sid,disabled` can show the SID on the computer, but it is different with what we get from token.

Reply:
That's interesting. Sorry I didn't see the internal design review first! Will continue discussion there.